src/main.rs

```diff
@@ -36,7 +36,14 @@ pub mod expansion;
 pub struct Command {
     pub name: &'static str,
     pub help: &'static str,
```

Comment:
You might want to make this a function, since it's not necessarily being static.